src/js/store/configureStore.js

```diff
@@ -0,0 +1,16 @@
+var createStore  = require('redux').createStore;
+var rootReducer = require('../reducers');
+
+var configureStore = function(initialState) {
+  var store = createStore(rootReducer, initialState);
+  if (module.hot) {
+    // Enable Webpack hot module replacement for reducers
+    module.hot.accept('../reducers', () => {
+      var nextReducer = require('../reducers').default;
+      store.replaceReducer(nextReducer)
+    })
```

Comment:
Part of me wants to avoid doing any of this until we are in a better position to utilize hot-swapping; my experience with discussing hot swap configurations with our colleagues suggests to me that it's just going to be noise for now